analyzer/analyzer.go

```diff
@@ -30,16 +32,20 @@ func (a *Analyzer) AnalyzeSchema(strOrPath string, includes []string, excludes [
 		return fmt.Errorf("failed to analyze schema: %w", err)
 	}
 
-	if err := s.Filter(&schema.FilterOption{
+	a.Schema = s
+
+	return nil
+}
+
+func (a *Analyzer) FilterSchema(includes []string, excludes []string, labels []string, distance int) error {
+	if err := a.Schema.Filter(&schema.FilterOption{
```

Comment:
I got your point. I will add some commits to address the issue. 👌

Reply:
Analyzer is in effect just creating Analyzer.Schema and filtering it.
If that is the case, it might be better to make it a pure Schema manipulation function.

In particular, I am concerned about the fact that this line is designed to perform destructive operations on the schema.

Reply:
AnalyzeSchema and FilterSchema are the same method, but we need to be careful about the order in which they are executed.

Reply:
Looks Nice!!